test/extended/registry/signature.go

```diff
@@ -0,0 +1,116 @@
+package registry
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	imagesutil "github.com/openshift/origin/test/extended/images"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("image signature workflow", func() {
+	defer g.GinkgoRecover()
+	var oc = exutil.NewCLI("registry-signing", exutil.KubeConfigPath())
+
+	g.It("can push a signed image to openshift registry and verify it", func() {
+		g.By("building an signer image that know how to sign images")
+		_, err := oc.Run("new-build").Args("--docker-image", "openshift/origin:latest", "--to", "signer:latest", "-D", "-").InputString(`FROM openshift/origin:latest
```

Comment:
you can do it, but not via new-build/new-app.  you'd have to define the build yourself using a dockerimage reference.